.gitignore

```diff
@@ -1,3 +1,6 @@
+# Docusaurus generated file
+docs/
+
```

Comment:
We don't want to do this (_yet_ or possibly ever). The `docs/` folder is the source that Docusaurus uses to render the site. We check in all of the pre-processed MarkDown files as this makes it much easier for us to use Netlify's preview environments. Netlify currently doesn't need to worry about running our pre-processor to dynamically source all of the content at compile time. For that matter, none of our other environments need to run the pre-processor right now either.

This aligns with our future approach of the pre-prorcessor running as a service that takes action in response to certain events (like GitHub webhooks). The pre-processor itself will be raising PRs that will then be approved and merged. 

Do you have a different take on it?